packages/gatsby-plugin-vercel-analytics/package.json

```diff
@@ -2,16 +2,12 @@
   "name": "@vercel/gatsby-plugin-vercel-analytics",
   "version": "1.0.10",
   "description": "Track Core Web Vitals in Gatsby projects with Vercel Speed Insights.",
-  "main": "index.js",
+  "main": "gatsby-browser.js",
```

Comment:
What happened to `index.js`?

Reply:
Nothing. Seems like this was making tests fail too 🤷 I reverted this part.